### ISSUE TITLE
slow recognition of key combination ctrl-a n / ctrl-a p

### ISSUE THREAD

**Issue by f1-outsourcing** [tmux/tmux#1252]
Wanted to give tmux a try, comming from screen. But I constantly have to do ctrl-a p/ctrl-a n twice because tmux did not catch the first one. And this is not my typing, if been using screen for years, very irritating first experience.

```
Linux os0 2.6.32-696.13.2.el6.x86_64 #1 SMP Thu Oct 5 21:22:16 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux


$ rpm -qa | grep tmux
tmux-1.6-3.el6.x86_64

~]$ cat .tmux.conf
# ctrl-a key
unbind C-b
set -g prefix C-a
bind C-a send-prefix

# No delay for escape key press
set -sg escape-time 0


```

**Comment by nicm** [comment]
I bet you are pressing C-a C-p not C-a p.


On 7 Feb 2018 9:19 am, "F1 Outsourcing Development" <
notifications@github.com> wrote:

Wanted to give tmux a try, comming from screen. But I constantly have to do
ctrl-a p/ctrl-a n twice because tmux did not catch the first one. And this
is not my typing, if been using screen for years, very irritating first
experience.

Linux os0 2.6.32-696.13.2.el6.x86_64 #1
<https://github.com/tmux/tmux/pull/1> SMP Thu Oct 5 21:22:16 UTC 2017
x86_64 x86_64 x86_64 GNU/Linux

$ rpm -qa | grep tmux
tmux-1.6-3.el6.x86_64

~]$ cat .tmux.conf
ctrl-a key

unbind C-b
set -g prefix C-a
bind C-a send-prefix
No delay for escape key press

set -sg escape-time 0

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/tmux/tmux/issues/1252>, or mute the thread

**Comment by f1-outsourcing** [comment]
I will take that bet, how much you want to put on it? Beware I have been using screen for 15+ year or so. ;)

**Comment by nicm** [comment]
screen and tmux are not the same.

**Comment by nicm** [comment]
If you are convinced you are not pressing C-a C-p (note that screen binds this by default and tmux doesn't), then please show me logs from reproducing the problem.

**Comment by nicm** [comment]
Actually, you need to upgrade tmux first, 1.6 is way too old.

**Comment by chros73** [comment]
It always worked for me (I also use prefix+a).

**Comment by f1-outsourcing** [comment]
> If you are convinced you are not pressing C-a C-p (note that screen binds this by default and tmux doesn't), then please show me logs from reproducing the problem.

Interesting, I did not know that. But why not learn from the experience that screen has, implement something similar? Al I notice I can faster Ctrl-a n in screen then in tmux. I guess screen has implemented it because there was a demand for it. 

> Actually, you need to upgrade tmux first, 1.6 is way too old.

I am sort of sticking to the default rpms of CentOS6/CentOS7. I installed tmux on a CentoOS7 host that was tmux-1.8-4.el7.x86_64. But is also 'slow' in switching
